datasets/modis-ndvi.data.mdx

```diff
@@ -0,0 +1,91 @@
+---
+id: MODIS_Terra_L3_NDVI_16Day
+name: MODIS Terra 16-Day NDVI
+featured: true
+description: The Vegetation Index (L3, 16-Day) layer is created from the Terra Moderate Resolution Imaging Spectroradiometer (MODIS) Vegetation Indices (MOD13Q1) data which are generated every 16 days at 250 meter (m) spatial resolution as a Level 3 product.
+media:
+  src: ::file ./media/modis-ndvi.jpg
+  alt: MODIS NDVI over the Eastern Corn Belt of the Midwestern USA.
+  author:
+    name: Andrew Blackford
+infoDescription: |
+  ::markdown
+    - Temporal Extent: 2000 Mar 5 - Present
+    - Temporal Resolution: Daily
+layers:
+  - id: MODIS_Terra_L3_NDVI_16Day
+    stacApiEndpoint: https://staging.openveda.cloud/api/stac
```

Comment:
```suggestion
```

This collection is now in the production catalog: https://openveda.cloud/api/stac/collections/MODIS_Terra_L3_NDVI_16Day